doc/api/stream.md

```diff
@@ -67,7 +74,7 @@ var server = http.createServer( (req, res) => {
   // the end event tells you that you have entire body
   req.on('end', () => {
     try {
-      var data = JSON.parse(body);
+      const data = JSON.parse(body);
     } catch (er) {
       // uh oh!  bad json!
       res.statusCode = 400;
```

Comment:
Definitely good with this but perhaps as a separate commit in this PR? Do you have a replacement example already in mind?